### PR TITLE
Ensure PRs are up-to-date and have passing checks

### PR DESCRIPTION
**Pull request by rtsao**

### CODE REVIEW

**Review by chrisdothtml — DISMISSED** [review]
The prettier config shouldn't be needed as long as you have an eslint editor plugin, but doesn't really matter to me

File: lib/lang.js [hunk 1]
```diff
@@ -65,6 +65,8 @@ module.exports = {
     `This pull request was landed via ${landedRepos[repoName].sha}.`,
   error_approval: command =>
     `Unable to ${command}. At least one approved review is required.`,
+  error_up_to_date_passing: command =>
```
Comment (chrisdothtml) [inline lib/lang.js]: tiny nit: the underscore is just after the type of comment it is, otherwise they're camel case

**Comment by rtsao** [comment]
> The prettier config shouldn't be needed as long as you have an eslint editor plugin, but doesn't really matter to me

Huh, for some reason even `yarn lint --fix` wasn't working for me without it...

**Comment by chrisdothtml** [comment]
Oh yeah, guess it's not working for me either. Not sure why not

**Review by chrisdothtml — APPROVED** [review]

**Pull request merged**